### ISSUE TITLE
Questions regarding “TP-Link Archer T3U Plus”

### ISSUE THREAD

**Issue by marwenius** [dividebysandwich/shareboxx#10]
The stick works in principle under “Raspberry Pi OS (64-bit)” (before and after installing the driver). I have installed the complete OS for better troubleshooting (at least for me).

I can connect with the shareboxx-SSID, but either no captive portal appears or when it tries to load it, it gives an error message regarding ngnix.

With the wlan0 option and without the stick plugged in, it works without any problems.

That is why I have the following questions:

Have you only changed wlan0 to wlan1 in the following files or have you changed something else?
```
/etc/dnsmasq.conf
/etc/dhcpcd.conf 
/etc/hostapd/hostapd.conf
```

Is the stick plugged into a USB 2.0 or USB 3.0 port?

What is the content of the following file? Which options are set (`options 88x2bu [...]`)?
```
/etc/modprobe.d/88x2bu.conf
```

Thank you in advance!

**Comment by dividebysandwich** [comment]
Hi, I think those are the only files changed, yes. I had mine plugged into a USB 3 port just in case.
I can't tell you what the contents of the file in modprobe is because my raspi with the original setup got lost in a flood - I am not joking, unfortunately. But I am pretty sure I did not change anything. What you could try is to just see if you can use hostap at all - maybe there's been a quiet hardware revision on the wifi adapter and it doesn't support AP mode anymore?

**Comment by maxxxxpower** [comment]
Yes I'd also like to know if anything else changes. *Does the driver information in /etc/hostapd/hostapd.conf need to change once I switch to wlan1? If so, what do I enter? 

EDIT: Following up - I did not have to change anything other than wlan0 to wlan1.

**Comment by dividebysandwich** [comment]
New setup script lets you select from the discovered interfaces, and even shows a warning if it's known not to support AP mode. Closing this one as manual config editing should not be required anymore.
